guides/common/modules/con_fapolicyd-on-server.adoc

```diff
@@ -0,0 +1,15 @@
+[id="fapolicyd-on-server_{context}"]
+ifeval::["{context}" == "{project-context}"]
+= Optional: fapolicyd on {ProjectServer}
```

Comment:
Also, the assembly title would work better with a verb. Like "Using `fapolicyd` on {ProjectServer}"? You could also just merge the content of this assembly ("2.4. Optional: fapolicyd on Satellite Server") into the procedure introduction ("2.4.1. Installing fapolicyd on Satellite Server").

Reply:
I am not sure if we are marking up in the title but definitely adding verb. :+1: 
Also, I would like to keep it separate from the procedure because the installation of Project Server is not yet completed.